wikitemplate-cr-upgrade.md

```diff
@@ -114,7 +115,15 @@ _Each start should take less than 7 seconds_
 
 ## Geolocation
 
-- [ ] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation works
+- [ ] Check that https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation is blocked due to cross origin iframes
+- [ ] Check that https://browserleaks.com/geo works and shows correct location
+- [ ] Check that  https://html5demos.com/geo/ works but need not be accurate location
```

Comment:
nitpick: "Check that https://html5demos.com/geo/ works but doesn't require an accurate location"

Seems like there's a space between `that` and the URL.